lib/jirahelper/issue.rb

```diff
@@ -13,8 +13,11 @@ def fetch_issue(key, expected = true)
     #
     # @param [Type String] jql Valid JQL query
     # @return [Type Array] 0-m JIRA Issues returned from query
-    def fetch_issues(jql)
+    def fetch_issues(jql, suppress_exceptions = false)
       client.Issue.jql(jql)
+    rescue => e
+      throw e unless suppress_exceptions
```

Comment:
This is to ensure there are no regressions for the `myissues` route: https://github.com/esigler/lita-jira/blob/master/lib/lita/handlers/jira.rb#L124